packages/hooks/src/useTransferMessageBetweenWalletAndTradingApp.ts

```diff
@@ -0,0 +1,74 @@
+import useExchangeRate from './useExchangeRate';
+import useWalletTransfer from './useWalletTransfer';
+
+type TTransferMessageBetweenWalletAndTradingApp =
```

Comment:
@aum-deriv maybe better to use common type with property `code: 'WalletToTradingAppDailyLimit' | 'DemoWalletToTradingAppDailyLimit'`, because the rest properties are equal